client/src/plus/deep-dives/survey/index.scss

```diff
@@ -77,8 +77,12 @@
   }
 
   .survey-success {
+    align-items: center;
+    display: flex;
     font-size: typography.$xl-font-size;
+    justify-content: center;
     line-height: 150%;
+    min-height: 250px;
```

Comment:
Shouldn't this have been defined on the parent?

Reply:
In case you have a 3rd survey page where there's only 1 question and it's thus smaller than both the previous page and the thank you page.